biostudiesclient/rest_error_exception.py

```diff
@@ -0,0 +1,18 @@
+"""
+biostudiesclient.rest_error_exception
```

Comment:
I believe the convention with python is to have a single file called `exceptions.py` that contains all the exceptions for a package.

Reply:
yeah, I am a Java guy. I still have to learn these specific conventions related to Python packages/modules.
As it only contained 1 exception I named it after that name.
I am going to rename it.

Reply:
Done and pushed